sdk/src/main/java/io/opentelemetry/sdk/resources/EnvVarResource.java

```diff
@@ -75,4 +78,35 @@ static Attributes parseResourceAttributes(@Nullable String rawEnvAttributes) {
       return attrBuilders.build();
     }
   }
+
+  /** Builder utility for this EnvVarResource. */
+  public static class Builder extends ConfigBuilder<Builder> {
```

Comment:
I'd suggest making this `private` (or package-protected if you need `@VisibleForTesting`). it doesn't seem like public builder is needed at this time at lease
```suggestion
  private static class Builder extends ConfigBuilder<Builder> {
```